vars/gh.groovy

```diff
@@ -37,12 +37,14 @@ def call(Map args = [:]) {
     def flagsCommand = ''
     if (flags) {
       flags.each { k, v ->
-        if (v instanceof List) {
-          v.each { value ->
-            flagsCommand += "--${k}=${value} "
+        if (v instanceof java.util.ArrayList || v instanceof List) {
+          v.findAll { it.trim() }.each { value ->
+            flagsCommand += "--${k}=\"${value}\" "
           }
         } else {
-          flagsCommand += "--${k}=${v} "
+          if (v?.trim()) {
+            flagsCommand += "--${k}=\"${v}\" "
+          }
```

Comment:
This will solve issues with spaces and only add the flag if the value is not empty